charmcraft.yaml

```
@@ -71,16 +78,8 @@ resources:
     filename: openstack-exporter.snap
     description: |
       Snap for openstack-exporter.
-      This is the software that implements actual metrics collection and endpoint for prometheus scraping.
-
-      The snap resource on Charmhub is intended to be an empty file.
-      The charm will ignore an empty file resource,
-      so that the resource can be used as an override by the user,
-      if they wish to provide a local snap.
-
-      Currently a local snap must be provided via the resource,
-      since the supported snap isn't published on the snap store.
-      When it is published, the charm will use the snap from the store,
-      and a user can provide a local snap to use instead.
-
-      This override may be useful for testing or in environments with network restrictions.
+      This is the software that implements actual metrics collection and endpoint for prometheus
+      scraping. The snap resource on Charmhub is intended to be an empty file. The charm will
+      ignore an empty file resource and install from the snap store, so that the resource can be
+      used as an override by the user, if they wish to provide a local snap or if it's an airgapped
+      environment where snap-proxy is not a possible solution.
```

rewording suggestion
```suggestion
      This is the software that implements actual metrics collection and an endpoint for prometheus
      scraping. The snap resource on Charmhub is intended to be an empty file. The charm will
      ignore an empty file resource and normally install the snap from the snap store, but a custom snap can also be provided here if needed.
```
(I would not mention snap-proxy not being a possible solution since it's supposed to be a possible solution :))